### PR TITLE
1170 frontend notes

### PR DESCRIPTION
**Pull request by eugenk**
This replaces #1203, because that one had the wrong destination branch.

##### Original message
- public/.htaccess is frontent specific (apache httpd, only) and thus
  is out of the application scope, belongs as a side note to docs/wiki
- see https://github.com/ontohub/ontohub/issues/1170

### CODE REVIEW

File: doc/frontend-notes.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Set the document root to ontohub's "public" directory (e.g.
+"DocumentRoot /local/home/ontohub/ontohub/public").
+
+Make sure, that the rewrite module is activated, enable the rewrite engine
+("RewriteEngine On"), and put the follwing two blocks before any proxy
+directives:
+
+	# Maintenance mode
+	RewriteCond /data/git/maintenance.txt -f
```
Comment (eugenk) [inline doc/frontend-notes.txt]: This should not be the data/git directory (I assume this is the directory of the git repositories), but the `data/` directory of ontohub.

File: doc/frontend-notes.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI}.gz -s
+	RewriteRule ^(.*\.(js|css))$ $1.gz [E=no-gzip,QSA,L]
+
+The first block just says, that if the file /data/git/maintenance.txt exists
+and the URL path of the request does not start with the given aka
+"not-used-by-ontohub-web-app" prefix, it answers with an service error status
+and the content of the file associated with the given URL path in the
+RewriteRule. Note that ontohub's git service uses a similar approach - it
+doesn't accept any command as long as ${git.data_dir}/maintenance.txt exist.
+The value for ${git.data_dir} can usually be found in config/settings.local.yml
```
Comment (eugenk) [inline doc/frontend-notes.txt]: `${git.data_dir}` should be `data_dir` (see comment above).

File: doc/frontend-notes.txt (hunk 1 above)
Comment (eugenk) [inline doc/frontend-notes.txt]: I mean this should be the directory specified in `settings.yml` at `paths.data` (after the last settings pull request is merged)

**Comment by ebolloff** [comment]
:+1:

**Pull request merged**